backend/api.py

```diff
@@ -65,26 +74,32 @@ async def health() -> str:
 
 # LLM Dependency injection
 def get_llm() -> OpenAI:
-    load_dotenv()
+    # load_dotenv()
     openai_api_key = os.getenv("OPENAI_API_KEY")
+    print(openai_api_key)
     return OpenAI(openai_api_key=openai_api_key, temperature=0.5, max_tokens=1024)
 
 # Endpoint to get a leetcode hint
-@app.post("/hint", response_model=HintResponse)
-async def get_hint(problem: HintRequest, llm: OpenAI = Depends(get_llm)):
+@app.post("/hint", response_model=str)
+async def get_hint(promptData: HintRequest):
     """
     Endpoint to get a hint for a specific problem.
-    Takes in a problem with name, description, and code, then gets a hint using the OpenAI API.
-    The hint is then sent back to the frontend.
+    
+    Takes in a problem with name, description, and code, then gets a hint 
```

Comment:
Curious if we need this, I feel like the code is self-documenting

Reply:
Yeah a lot of the comments could be stripped out. Will keep the doc strings for http://localhost:8000/docs